strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/security/ldap/LdapAuthenticatorConfigurationControllerTest.java

```diff
@@ -198,7 +196,7 @@ public void ldapConfigurationTestRequiresValidUrl()
                .body("errors[0]['name']", equalTo("configuration.url"))
                .body("errors[0]['messages'][0]", equalTo("must be a valid URI"));
     }
-
+	@Disabled
```

Comment:
Why was this test disabled?

Reply:
A problem occurs with this test file and to keep it in line with the backport the IOException was removed and disabled was added.

Reply:
Did I break it (was it commented out in my pull), or does it only fail for you?
You would have to provide some more details regarding "what fails".

Reply:
The error ran into was:
Tests run: 15, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 29.202 s <<< FAILURE! - in org.carlspring.strongbox.controllers.configuration.security.ldap.LdapAuthenticatorConfigurationControllerTest
ldapConfigurationTestShouldFailWithInvalidConfiguration  Time elapsed: 0.339 s  <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Expected status code <200> but was <500>.

	at org.carlspring.strongbox.controllers.configuration.security.ldap.LdapAuthenticatorConfigurationControllerTest.ldapConfigurationTestShouldFailWithInvalidConfiguration(LdapAuthenticatorConfigurationControllerTest.java:225)

Then I found I didn't have this class set up in PR-1783. The changes made here are to match that PR.

Reply:
Is that the whole stacktrace of the exception?
Does this fail in your IDE, or in your terminal, or both?

Reply:
It failed in the terminal. Once I made the change to match the change made to it in PR-1783 everything worked fine.

Reply:
Does this mean you've removed the `@Disabled` part and re-enabled the test? This test needs to be enabled and looked into.

@madhukarbharti / @luan-cestari : Would you like to help @mknutsen79 investigate this failing test?

Reply:
I haven't gotten to removing the disabled and testing the test yet, been having to do some other things. The plan is to get on that this coming week if i'm not beaten to it yet.

Reply:
Any update on this? :)

Reply:
Yes, the latest commit should have the change made to the test. I think I found what the issue was in the test and made the change.

Reply:
Great! Did you push the code, after you re-enabled the test?

Reply:
Yes I did, it is in this latest pull request that I did on monday.

Reply:
I'm sorry... which one is that and do we then need to close this one instead?

Reply:
It should still be part of this request, its the latest commit i've done on this branch. Sorry for the bad wording there.